_projects/0600_ditap.md

```diff
@@ -0,0 +1,50 @@
+---
+# Page template info (DO NOT EDIT)
+layout: default
+banner_file: banner--people-projects-lg.svg
+banner_file_mobile: banner--people-projects-sm.svg
+project_page: true
+
+# Carousel (Edit this)
+carousel_title: "Improving how the government buys technology"
+carousel_summary: "As the pace of innovation has accelerated, the government’s ability to meet people’s expectations of functional digital solutions has not kept pace. To meet this demand, we created an immersive development and training program to improve digital services acquisition expertise across government agencies.
+"
+carousel_image_name: project-ditap-card.jpg
+
+# Project detail page (Edit this)
+title: "Digital IT Acquisition Professional Program (DITAP)"
+agency: Cross-agency
+permalink: projects/ditap
+project_url: 
+
+# Impact statement (Edit this)
+impact_statement:
+  - figure: "400"
+    unit: "+"
+    description: |-
+      People will have graduated from our DITAP program by end of FY 2020
+
+  - figure: 4
+    unit: "x"
+    description: |-
+      Growth of the Digital Acquisition community since 2016, demonstrating the scale that USDS and OFPP envisioned when the program was created
+
+---
+
+## The Challenge
+
+Current federal procurement practices rarely provide the flexibility required to buy and deliver modern digital services. Government’s use of digital solutions has not kept pace with peoples’ expectations. As the pace of innovation has accelerated, the government’s ability to meet people’s expectations of functional digital solutions has not kept pace. To meet this demand, the Office of Federal Procurement Policy (OFPP) and USDS needed an immersive development and training program to improve digital services acquisition expertise across government agencies.
+
+## The Solution
+
+In 2016, USDS and OFPP created the Digital IT Acquisition Professional Program (DITAP), a Federal Acquisition Certification in Contracting (FAC‑C) certification program that trains contracting professionals to execute digital service procurements by acting as business advisors to agencies seeking to buy better digital services. DITAP leverages flexibilities in the Federal Acquisition Regulation to better acquire today’s technology and works seamlessly with digital service teams or acquisition innovation labs. Graduates of the DITAP program can also lead agency training workshops and provide consultations, expanding digital service procurement expertise across the government. In a [May 2018 memo](whitehouse.gov/wp-content/uploads/2018/05/fac_c_digital_services_05_12_18.pdf), OMB determined that beginning in FY 2022, contracting professionals assigned to digital services acquisitions above $7 million will be required to have FAC‑C‑DS certification.
```

Comment:
```suggestion
In 2016, USDS and OFPP created the Digital IT Acquisition Professional Program (DITAP), a Federal Acquisition Certification in Contracting (FAC‑C) certification program that trains contracting professionals to execute digital service procurements by acting as business advisors to agencies seeking to buy better digital services. DITAP leverages flexibilities in the Federal Acquisition Regulation to better acquire today’s technology and works seamlessly with digital service teams or acquisition innovation labs. Graduates of the DITAP program can also lead agency training workshops and provide consultations, expanding digital service procurement expertise across the government. In a [May 2018 memo](https://whitehouse.gov/wp-content/uploads/2018/05/fac_c_digital_services_05_12_18.pdf), OMB determined that beginning in FY 2022, contracting professionals assigned to digital services acquisitions above $7 million will be required to have FAC‑C‑DS certification.
```
(updated the url to include `https://` prefix; else, it'll be treated like a relative url on the usds.gov website)